scheduler/src/cook/mesos/scheduler.clj

```diff
@@ -14,47 +14,46 @@
 ;; limitations under the License.
 ;;
 (ns cook.mesos.scheduler
-  (:require [mesomatic.scheduler :as mesos]
-            [mesomatic.types :as mtypes]
+  (:require [chime :refer [chime-at chime-ch]]
```

Comment:
Let's get a pass on this file as it has already been done. I will create a separate PR for all the other files together.